website/docs/reference/node-selection/state-comparison-caveats.md

```diff
@@ -45,6 +45,21 @@ If you're a frequent user of `relationships` tests or data tests, or frequently
 dbt run -s "state:modified"
 dbt test -s "state:modified" --exclude "test_name:relationships"
 ```
+### Manifest overwritten in subsequent parses
+
+dbt now overwrites the `manifest.json` file during parsing. This means, if you're referencing `--state` information from the `target/ directory`, you may encounter a warning indicating that the saved manifest was not found, since the `manifest.json` is overwritten during parsing.
```

Comment:
since this was introduced in 1.5, it isn't really a 'now' change?
```suggestion
dbt overwrites the `manifest.json` file during parsing, which means when you reference`--state` from the `target/ directory`, you may encounter a warning indicating that the saved manifest wasn't found.
```